focus-group.js

```diff
@@ -150,11 +150,23 @@ export function focusGroupKeyUX(options) {
           inGroup = true
           window.addEventListener('keydown', keyDown)
         }
-        let items = getItems(event.target, group)
-        for (let item of items) {
-          if (item !== event.target) {
-            item.setAttribute('tabindex', -1)
-          }
+
+        let items = Array.from(getItems(event.target, group)).filter(
+          item => !item.closest('[focusgroup="none"]')
+        )
+        if (!items.length) return stop()
```

Comment:
Do we really need it? We have `forEach` later. On empty array they will do nothing.

`items[0].focus()` → `items[0]?.focus()`